src/cloud/docker/docker-mode-production.md

```diff
@@ -11,28 +11,14 @@ Production mode is the default configuration setting for launching the Docker en
 
 {%include cloud/note-docker-config-reference-link.md%}
 
-{:.procedure}
-To launch the Docker environment in production mode:
-
-1. Download a Magento application template from the [Magento Cloud repository][cloud-repo] and copy the files to your Magento Cloud Docker project directory. Be careful to select the branch that corresponds with the Magento version.
-
-1. Add your [Magento access credentials][magento-creds] to the `auth.json` file.
-
-1. Run the installation script, `init-docker.sh` to install the template dependencies and add the default hostname to your `/etc/hosts` file.
-
-   ```bash
-   curl https://raw.githubusercontent.com/magento/magento-cloud-docker/<magento-cloud-docker-package-version>/bin/init-docker.sh | bash
-   ```
-
-   For `<package-version>`, use the [latest release of the {{site.data.var.mcd-package}}].
+## Prerequisites
```

Comment:
H2 seems odd for prerequisites because then the procedure becomes a subset. I suppose Bolding is not preferred style for the linter...but not sure what is. Maybe procedure class again...

Reply:
Fixed locally.